ci/test/pipeline.template.yml

```diff
@@ -716,7 +716,20 @@ steps:
       branch: "$BUILDKITE_BRANCH"
       env:
         BUILDKITE_TAG: "$BUILDKITE_TAG"
-    if: build.tag != "" && build.branch =~ /^v/
+    if: build.tag != "" && build.branch =~ /^v\.v/
+    coverage: skip
+
+  - id: nightly
```

Comment:
This needs to be named "nightly-if-release" and also given a more informative label + comment. We do not want people casually reading the yml to come to conclusion that Nightly will be auto-triggered unless in this very specific circumstance.